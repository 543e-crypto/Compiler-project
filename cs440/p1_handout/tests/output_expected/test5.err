Token Summary:
   Keywords: 22
   Punctuations: 13
   Operators: 2
   Comments: 5
   Numbers: 3
   Strings: 0
   IDs: 14
