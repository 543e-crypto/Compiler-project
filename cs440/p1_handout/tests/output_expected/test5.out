Special IDs: foo bar me once 
